README.md

```diff
@@ -1,2 +1,49 @@
+# Introduce junbae
+
+We introduce Junbae for people using the this vscode extenstion.
+We don't know the gender yet, but i think it will be a pretty cute parrot who will be raised by the person making this program.
+The name is taken from Junbae's name.
```

Comment:
```suggestion
We introduce Junbae for people using this vscode extension.
We don't know the gender yet, but I think it will be a pretty cute parrot who will be raised by the person making this program.
The name is taken from Junbae's name.
```